Included a link to Application Security Group template example

paulomarquesc opened the pull request:
Hi, last week I published an Azure Quickstart template that shows how to apply Application Security Groups and may be a good idea to add a link in the documentation so customers can see how it is done in templates since we have an example on how to do it from Azure CLI and Powershell.

azurecla commented:
Hi __@grnadav__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@grnadav : Thanks for your contribution to the Azure documentation! The author, @azdocprs, has been notified to review your proposed change.

tysonn commented:
@telmosampaio - Telmo, could you please review the proposed update? This include is used only in one of your articles.

telmosampaio commented:
#sign-off

paulomarquesc committed:
Included a link to Application Security Group template example

Hi, last week I published an Azure Quickstart template that shows how to apply Application Security Groups and may be a good idea to add a link in the documentation so customers can see how it is done in templates since we have an example on how to do it from Azure CLI and Powershell.

paulomarquesc committed:
Merge pull request #1 from paulomarquesc/ASG-template

Included a link to Application Security Group template example
